OpenOversight/app/main/views.py

```diff
@@ -432,8 +432,12 @@ def submit_complaint():
 @main.route('/submit', methods=['GET', 'POST'])
 @limiter.limit('5/minute')
 def submit_data():
-    departments = Department.query.all()
-    return render_template('submit_deptselect.html', departments=departments)
+    if User.dept_pref is not None:
```

Comment:
I see that my guess on how to get data from the db was wrong. Went looking for something similar. Maybe `if User.query.filter_by(dept_pref=field.data).first():`?